frontend/src/concepts/pipelines/content/createRun/useRunFormData.ts

```diff
@@ -30,7 +30,7 @@ import {
 } from '~/concepts/pipelines/content/createRun/const';
 import { convertDateToTimeString } from '~/utilities/time';
 
-const isPipelineRunJob = (
+export const isPipelineRunJob = (
```

Comment:
This will probably need to be moved to another location -- I was lazy my first pass and they are not intended to be exported. 

You'll want to rebase onto the feature branch before doing that, there are two of them.